site/en/docs/webstore/best_practices/index.md

```diff
@@ -45,24 +45,19 @@ standards of performance, security, and user experience, as summarized by the fo
 
 ## Support Google Accounts
 
-If your app requires user login, we recommend that you provide at least some support for Google
-Accounts. The reason is that if a user purchases your app from the Chrome Web Store, odds are the
-user is already logged into a Google Account whenever they use your app. Reducing the number of
+If your extension requires user login, we recommend that you provide at least some support for Google
+Accounts. Google Accounts can help you provide a better user experience, since users of the Chrome
+Web Store are likely to be logged in already, and they won't have to set up and remember yet another
+username and password. Reducing the number of
 logins improves the user's experience.
 
-If you already have a login system, consider correlating the Google Account ID to the user account
-in your system. You can do this by storing the user's OpenID URL from Google's OpenID service, in
-the same way that you store other data that's associated with a user's account. When someone is
-logged into a Google Account but not your login system, you can automatically log them into their
-user account in your system.
```

Comment:
This strikes me as a useful recommendation. Even if a developer didn't want to use a Google Account as their primary authentication method, it may still be useful and desirable to allow users to log in with their Google Accounts.

@AmySteam, could you comment on why you'd like to remove this section?

Reply:
@dotproto Good question!

### Background

I removed this paragraph because it strikes me as misleading and ambiguous. The first sentence seems promising, but the following sentences are not helpful. Let me explain.

>  You can do this by storing the **user's OpenID URL** from Google's OpenID service, in the same way that you store other data that's associated with a user's account.

This refers to a previous flow for openid. Currently the OAuth2 flow does not return an **OpenID URL**. Unless this is referring to the redirect URL that contains the OpenID token. 🤔 For more context, see [Retrieve a redirect URL](https://deploy-preview-1757--developer-chrome-com.netlify.app/docs/webstore/identify_user/#redirect-url).

> When someone is **logged into a Google Account** but not your login system, you can **automatically log them into their user account** in your system.

- The OpenID OAuth2 flow requires `chrome.identity.launchWebAuthFlow()`, which launches a consent screen.
- You can automatically access the Google account that the user is already logged into with `chrome.identity.getAuthToken()`. See [OAuth2: Authenticate users with Google](https://developer.chrome.com/docs/extensions/mv3/tut_oauth/#identity_permission)

### Options

I think we can either remove this phrase or re-write it. Something like:

If you already have a login system, consider correlating the Google Account ID to the user account in your system. You can use the Chrome Identity API to support Google accounts in the following ways:
 
- Google OAuth2/OpenID: See [Identifying the User](https://deploy-preview-1757--developer-chrome-com.netlify.app/docs/webstore/identify_user).
- OAuth2: See [Authenticate users with Google](https://developer.chrome.com/docs/extensions/mv3/tut_oauth/#identity_permission).

I'm open to suggestions 🙂

Reply:
Let me know when you both have resolved this and I'll review the text.

Reply:
@AmySteam, I like the suggestion to rephrase it. Let's run with that.